validium_mode_example/src/main.rs

```diff
@@ -0,0 +1,275 @@
+use std::{str::FromStr, time::Duration};
```

Comment:
The core monorepo follows a certain hierarchical structure where all the Rust code is located in the `core` directory, and adding a crate to the root seems to be inconsistent.

I'm not sure what the right place for this would be, but I guess it can be explained by the fact that this crate has no concrete purpose. I assume that it's useful during the development for debugging/testing, and serves a good example of usage, but what would the use cases be after this PR is merged?

I'm not sure if people would often try to run this example. At the same time, the developers of the core monorepo would have to maintain it to avoid having dead/outdated code.

Overall, I think that it'd be better to either remove this crate or repurpose it to be an integration test, so that it's run continously to check the validium mode logic.